### DISCUSSION TITLE
How to re-enable notifications in Windwos 11?

### DISCUSSION THREAD

**Discussion by stax76** [Flow-Launcher/Flow.Launcher#946] [Q&A] [answered]
I disabled Flow Launcher notifications system-wide in Windows 11, I think I did this with a UI setting, and now I'm not able to re-enable notifications, I'm not getting a notification when I press F5 using v1.9.1. I've searched the Windows 11 settings UI and the registry but could not find a way to be re-enabled notifications. In the Windows settings under 'System > Notifications' Flow Launcher is not listed, I tried reinstalling, but it did not help.

**Comment by taooceros** [comment]
Have you enabled focus assist?

**Reply by taooceros** [reply]
I think flow doesn't register the notification so it does not show in the list. It's interesting how you disable it.🤔

**Reply by stax76** [reply]
Focus assist is disabled. I think I know now how I disabled it. In the notification popup or in the notification when displayed in the notification  center, there is a hamburger menu, that's where it was disabled.

Monitoring the explorer process with Process Monitor to find out which registry keys are written might work, or I ask in a forum that is specialized for this kind of topic.

**Comment by onesounds** [comment]
I tested and confirmed that there was a problem. I'm looking for a solution.

**Comment by taooceros** [comment, marked as answer]
I think this is resolved in the latest release? Whenever flow is opened, a notification option for it will be created.
